lib/repositories/activity/good_activity_repository.dart

```diff
@@ -0,0 +1,27 @@
+import 'dart:developer';
+
+import 'package:application/models/activity.dart';
+import 'package:application/repositories/activity/i_activity_repository.dart';
+import 'package:application/repositories/search/search_repository.dart';
+import 'package:cloud_firestore/cloud_firestore.dart';
+
+import '../rest_api.dart';
+
+class GoodActivityRepository implements IActivityRepository {
```

Comment:
I think the name should be more representative of how this repository uses the Flask backen or how it's different to the existing one.